generators/app/templates/webpack.config.js

```diff
@@ -23,6 +23,7 @@ if (process.env.INCLUDE_WEBPACK_HTML) {
 
 module.exports = {
   entry: [
+    <%- withFetch ? "'whatwg-fetch'," : '' %>
```

Comment:
It does. Right now it leaves a blank whitespace, which doesn't interfere with anything running. However I couldn't figure out with ejs how to get rid of the whitespace if we don't have the variable. There is a closing tag `-%>` that trims the next newline I think. But I can't really do that conditionally?